lib/gui/components/update-notifier.js

```diff
@@ -138,17 +166,48 @@ module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_DAYS, M
    */
   this.notify = () => {
     return this.getLatestVersion().then((version) => {
-      return ModalService.open({
-        template: './components/update-notifier/templates/update-notifier-modal.tpl.html',
-        controller: 'UpdateNotifierController as modal',
-        size: 'update-notifier',
-        resolve: {
-          options: _.constant({
-            version
-          })
+      settings.set('lastUpdateNotify', Date.now());
+      settings.set('sleepUpdateCheck', false);
```

Comment:
I think I can say what @Shou is saying, although without thinking about it in more depth (which I don't have time for right now) I can't 100% follow the code-arguments from either @Shou or @jviotti 

I _think_ @Shou is saying that it only makes sense to update `lastUpdateNotify` **if** the user ticked "remind me again in 7 days". If the user doesn't tick the box, then although we _could_ update `lastUpdateNotify` there's not strictly any need to. Even @jviotti mentioned above "(note that the lastUpdateNotify is not even considered, given that !settings.get('sleepUpdateCheck') is already true)" - I guess @Shou was just taking that one step further.